dali/python/bundle-wheel.sh

```diff
@@ -44,6 +44,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 INWHL=$(readlink -e $1)
+PATH_PREFIX=${2:-local}
```

Comment:
```suggestion
PATH_PREFIX=${2:-/usr/local}
```

Reply:
If it's a prefix it should be a prefix :)

Reply:
Changed to `LIB_PATH`

Reply:
Done